website/docs/r/route_table.html.markdown

```diff
@@ -85,8 +85,19 @@ attribute once the route resource is created.
 
 ## Import
 
-Route Tables can be imported using the `route table id`, e.g.
+~> **NOTE on Importing Routes:** Currently, importing an `aws_route_table` and then
```

Comment:
The two new paragraphs here seem wordy as well. Instead, we should probably just state the potential problem, its workaround, and the upcoming change in a single callout box:

> **NOTE:** Importing this resource currently imports an `aws_route_table` resource and `aws_route` resources for each route. If you intend to manage routes within the `aws_route_table` resource (using the `route` argument), you must use `terraform state rm` to delete the imported `aws_route` resources or Terraform will delete the routes. The behavior of importing `aws_route` resources with the `aws_route_table` resource will be removed in the next major version.